packages/nettacker/PKGBUILD

```diff
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=nettacker
+pkgver=0.0.2.r0.g92239ce
+pkgrel=1
+pkgdesc='Automate information gathering, vulnerability scanning and eventually generating a report for networks, including services, bugs, vulnerabilities, misconfigurations, and other information.'
```

Comment:
Please try to keep the description below 80 character in length. Keep it short and precise